metalog.js

```diff
@@ -89,7 +92,7 @@ class Console {
     try {
       console.assert(assertion, ...args);
     } catch (err) {
-      this._write('error', `${this._groupIndent}${err.stack}`);
+      this._write('error', this._groupIndent, err.stack);
```

Comment:
Because the formatter has to decide what to do with this indent. (i.e. json - ignore, pretty - indent everything).
This also allows to properly indent multiline messages if we need it later.

Reply:
Dropping indentation will be ok, but as a separate PR :smile:

Reply:
Do we need to pass it as a parameter? Maybe we can read it from property?

Reply:
Actually yes, we can add it in `this._write`.

Reply:
Though we didn't have "indent" for `count` and `table` methods and this change will add them there.